### PR TITLE
New project architecture

### PR DESCRIPTION
**Pull request by laugustyniak**

### CODE REVIEW

File: test/devTest.py [hunk 1]
```diff
@@ -1,14 +1,126 @@
-import model.parameters as model
-import data.Events as events
+import sys
+from uuid import getnode as get_mac
+if get_mac() == 2485377892363:
```
Comment (laugustyniak) [inline test/devTest.py]: we must talk about that f2f

File: data/data.py [hunk 2]
```diff
@@ -0,0 +1,320 @@
+import pandas as pd
+import numpy as np
+from collections import defaultdict
+import networkx as nx
+import pickle
+import itertools
+import functools
+
+
+class data():
```
[the remark this answers is not in the document]
Comment (macio232) [reply data/data.py]: Done

File: data/data.py [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+    def __init__(self):
+        self.eventLog = None
+        self.edges = None
+        self.numUsers = None
```
[the remark this answers is not in the document]
Comment (macio232) [reply data/data.py]: Done

File: data/data.py [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+    # review
+
+    def sortData(self):
+        if 'contagionID' in self.eventLog.columns:
+            self.eventLog.sort_values(by=['contagionID', 'ts'], inplace=True)
+        else:
+            self.eventLog.sort_values(by=['contagion', 'ts'], inplace=True)
+
+    def loadData(self, directory=None, eventLogDF=None, edgesDF=None, fileNames = ('eventLog','edges')):
+        ''' Loads data to class data instance from the source that depends on given arguments'''
```
[the remark this answers is not in the document]
Comment (macio232) [reply data/data.py]: Done

File: data/data.py [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+        self.numEvents = None
+        self.contagionIDDict = None
+        self.graph = None
+
+    def addGraph(self):
+        if self.graph is None:
+            self.graph = nx.from_pandas_edgelist(self.edges, 'user1', 'user2')
+
+    def loadDataFile(self, directory, fileNames = ('eventLog','edges')):
+        eventLogDF = pd.read_csv(directory + fileNames[0])
```
[the remark this answers is not in the document]
Comment (macio232) [reply data/data.py]: Done

**Pull request merged**